pom.xml

```diff
@@ -983,7 +983,11 @@
 					    <name>applicationGlobalConsoleDisplayName</name>
 					    <value>${application.globalconsole.display.name}</value>
 					</parameter>
-                                        <parameter>
+					<parameter>
```

Comment:
I'm not sure why you pass this parameter here.  This plugin is concerned with the production of the license.html file.